### ISSUE TITLE
Incorrect transformation of "srcset" attributes where URL contain commas

### ISSUE THREAD

**Issue by robertknight** [webrecorder/pywb#238]
Rewriting of "srcset" attributes splits the input value on commas (see https://github.com/ikreymer/pywb/blob/03b7cb4f28d523a838338c52f941bb7c21e90bc6/pywb/rewrite/html_rewriter.py#L237). However, this is not quite smart enough as URLs may contain commas in the path section (where they are "sub-delims") without being percent encoded.

This means that rewriting breaks URLs in srcset attribute values that contain commas. For example:

```html
<img src="https://iiif.elifesciences.org/journal-cms:content/2017-01/credit1.png/full/538,/0/default.jpg"
alt="Former choice options for author contributions"
srcset="https://iiif.elifesciences.org/journal-cms:content/2017-01/credit1.png/full/1076,/0/default.jpg 1076w"
class="captioned-asset__image">
```

The relevant part of the HTML specs are here: https://html.spec.whatwg.org/multipage/images.html#image-candidate-string . The only limitation on URLs in this context is that they cannot start or end with a comma.

**Comment by robertknight** [comment]
Unfortunately parsing srcset is not entirely trivial - https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/html/parser/HTMLSrcsetParser.cpp 😭

**Comment by ikreymer** [comment]
@robertknight Should now be fixed by #269

**Comment by ikreymer** [comment]
Fixed in pywb 2.0.0!
